Automattic/wp-calypso
# Gutenboarding site setup: use selected features to recommend plan

#### Changes proposed in this Pull Request

* Get `selected_features` from site options (see https://github.com/Automattic/wp-calypso/pull/45578 which added this property) and recommend a plan during Site Setup flow.

#### Testing instructions

* Go to [/new](https://calypso.live/new?branch=add/gutenboarding-site-setup-selected-features).
* Select some features on `/new/features` step and create the site.
* Apply the diff and sandbox the site.
* Start Site setup flow in editor and on Select a plan step, _Recommended plan_ should be displayed.

#### Screenshot
<img width="1405" alt="Screenshot 2020-09-17 at 11 30 14" src="https://user-images.githubusercontent.com/14192054/93446487-f75ed380-f8d9-11ea-85f2-f6267060594f.png">


Fixes #45148
